### ISSUE TITLE
Add support for maxHeapSize property on AsciidoctorTask

### ISSUE THREAD

**Issue by fossamagna** [asciidoctor/asciidoctor-gradle-plugin#267]
I have a project that using asciidoctor-gradle-plugin. It is built on CircleCI. It failed to build and I got exit code 137. I would like to specify max heap size to asciidoctor java process becouse container on CircleCI has usually memory RAM 4GB.
Please add support for maxHeapSize property on AsciidoctorTask.

**Comment by ysb33r** [comment]
It is available on the 2.0-alpha.3 if you use the new task type.

```groovy
plugins {
  id 'org.asciidoctor.jvm.convert' version '2.0-alpha.3'
}

asciidoctor {
    forkOptions {
       maxHeapSize '4096mb'
    }
}
```

Thre is no plans at present to backport this to 1.5.x.

**Comment by fossamagna** [comment]
I will try later and let you know what happened, thanks!

**Comment by ysb33r** [comment]
Closing issue that were fixed prior to 2.0 final release.
